### ISSUE TITLE
Multipath Control Application on Nexus 5 Phone

### ISSUE THREAD

**Issue by BlessedL** [MPTCP-smartphone-thesis/MultipathControl#2]
Greetings All,
I am using MPTCP on an Android Nexus 5 Phone and I am using the Multipath Control Application (specified on https://github.com/MPTCP-smartphone-thesis/MultipathControl) to control the settings and I had just had a few questions (would like to verify if my assumptions are correct) about how MPTCP should operate under the following Multipath Control App configurations. 
 
When Cellular is NOT SET AS BACKUP, and both WiFi and Cellular interfaces are ENABLED (via the Multi-Interface enabled option) and Cellular is NOT SET AS THE DEFAULT ROUTE does this mean the cellular interface IS STILL USED or IS IT NOT USED? Does this mean only WiFi Should be used? Below is an annotated version of the configuration I am talking about.
 
Multi-Interface is Enabled+
Default Route via cellular is off+
Set cellular as backup  is off +
= USE ONLY WiFi for this MPTCP Connection, is this assumption correct?
 
I have the same question for the following configurations as well
 
Multi-Interface Enabled+
Default Route via cellular is on+
Set cellular as backup off+ 
= Set cellular as primary path and WiFi as backup path or non primary path, is this assumption correct? What path/subflow would the default Min RTT scheduler select if both Cellular and WiFi both had space in it’s congestion window, but WiFi had a lower RTT than Cellular? Would it select the WiFi path  for this MPTCP Connection, is this assumption correct?
 
Multi-Interface Enabled+
Default Route via cellular is off+
Set cellular as backup on+ 
= Set cellular as backup path (non primary path) and WiFi as primary path is this assumption correct? What path/subflow would the default Min RTT scheduler select if both Cellular and WiFi both had space in it’s congestion window, but Cellular had a lower RTT than WiFi? Would it select the Cellular path  for this MPTCP Connection, is this assumption correct?
 
When the cellular interface is SET AS BACKUP, does this mean the cellular interface/path is only utilized when the WiFi interface/path is not available for use? What if the Cellular Interface has a lower RTT than the WiFi Interface and has space in it’s congestion window, would the MPTCP Default Min RTT Scheduler select the Cellular interface (the backup or non-primary interface) to send the next segment through even if the WiFi interface is also available (has space in it’s congestion window)? 

Basically, when MPTCP is using a Primary Path (let’s say WiFi) and a Non Primary Path (let’s say Cellular) and the default Min RTT scheduler, how does the scheduler select or use the Non Primary Path? 
 
Thank you for your reply
